ciphers/shuffled_shift_cipher.py

```diff
@@ -65,7 +52,7 @@ def __make_one_digit(self, digit):
                  else, converts to single digit and returns
```

Comment:
Please add the following doctests INSIDE the docstring of this function.  Once the Travis tests pass, we can land this one.
```
>>> ssc = ShuffledShiftCipher()
>>> ssc.__make_one_digit(1234)
10
>>> ssc.__make_one_digit(12345)
5
>>> ssc.__make_one_digit(123456)
3
```

Reply:
Note: 1234 is NOT returning a single digit.

Reply:
I'm currently writing such doctests and type hints. Thanks for the suggestions. Will commit soon.

Reply:
The problem is I just noticed this function isn't used anywhere anymore in the code. :no_mouth: I HAD used it in some previous version and it has no use in this one. :sweat_smile: 

I'll make the necessary changes and write some new doctests :man_facepalming:

Reply:
- [x] Removed the unnecessary function __make_one_digit()
- [x] wrote doctests for encrypt() and decrypt()
- [x] type hints introduced